core/state_processor.go

```diff
@@ -274,6 +284,26 @@ func applyTransaction(msg types.Message, config *params.ChainConfig, bc ChainCon
 	if err != nil {
 		return nil, err
 	}
+	var ETXRCount int
```

Comment:
camel case maybe or all caps ?

Reply:
I did this to reflect how its done everywhere else in the codebase (chameleon coding :lizard:). e.g. `ETXGas`, `ETXCache`, etc

I agree camel case would be better, but we should be consistent, and I don't want to change everywhere else in this PR